docs/host/host.md

```diff
@@ -139,27 +139,28 @@ kubectl drain <node_name> --force --ignore-daemonsets --delete-local-data --pod-
 Again, removing a control plane node in this situation is **not recommended** because etcd data is not replicated. Failure of a single node can cause etcd to lose its quorum and shut the cluster down.
 :::
 
-### 6. Remove the node.
-
-1. On the Harvester UI, go to the **Hosts** screen.
-
-1. Locate the node that you want to remove, and then click **⋮ > Delete**.
-
-![delete.png](/img/v1.2/host/delete-node.png)
-
-### 7. Delete RKE2 services on the node.
+### 6. Delete RKE2 services on the node and Shutdown the node.
 
 1. Log in to the node using the root account.
 
 1. Run the script `/opt/rke2/bin/rke2-uninstall.sh`.
 
+1. Shutdown the node.
```

Comment:
```suggestion
1. Run the script `/opt/rke2/bin/rke2-uninstall.sh` to delete RKE2 services running on the node.

1. Shut down the node.
```

Reply:
Out of curiosity, does the script only delete RKE2 services? 

I'm asking because the [RKE2 doc](https://docs.rke2.io/install/uninstall?_highlight=delete#rpm-method) has a more comprehensive description of what the script does. However, I also found [this issue](https://github.com/harvester/harvester/issues/2251). Does this mean the RKE2 doc's description does not apply to Harvester?

Reply:
From RKE2 source code, I think it disables RKE2 services and remove related file. Probably we need to revisit https://github.com/harvester/harvester/issues/2251.

https://github.com/rancher/rke2/blob/5b443424122858ee513bbc93101d1dd992e41f02/bundle/bin/rke2-uninstall.sh#L107-L113

No matter whether RKE2 files are removed, we run the script to prevent node from joining again.